### DISCUSSION TITLE
Another great collection of wildcards

### DISCUSSION THREAD

**Discussion by Jonseed** [adieyal/sd-dynamic-prompts#160] [General]
Just saw this posted:
[https://github.com/devilkkw/StableDiffusion/tree/main/wildcards](https://github.com/devilkkw/StableDiffusion/tree/main/wildcards)

**Comment by adieyal** [comment]
Great find. I'll link to it as another collection rather than steal their hard work.

**Comment by adieyal** [comment]
https://github.com/adieyal/sd-dynamic-prompts/tree/main/collections/devilkkw

**Comment by adieyal** [comment]
@devilkkw gave me permission to include those wildcards in the project so you can find the collection here: https://github.com/adieyal/sd-dynamic-prompts/tree/main/collections/devilkkw
